tests/db_migrator_test.py

```diff
@@ -674,4 +674,5 @@ def test_fast_reboot_upgrade_to_4_0_3(self):
         dbmgtr = db_migrator.DBMigrator(None)
         dbmgtr.migrate()
         expected_db = self.mock_dedicated_config_db(db_after_migrate)
-        assert not self.check_config_db(dbmgtr.configDB, expected_db.cfgdb)
\ No newline at end of file
+        assert not self.check_config_db(dbmgtr.configDB, expected_db.cfgdb)
+        assert dbmgtr.CURRENT_VERSION == expected_db.cfgdb.get_entry('VERSIONS', 'DATABASE')['VERSION']
```

Comment:
I think we need a dedicated test just for checking this assertion.

My goal is to use UT to always catch the issue that this PR is addressing -- missing update to CURRENT_VERSION

Reply:
I don't see how to achieve that, because not every UT mocks the config_db with a `VERSIONS` key

Reply:
I see. May be we should always verify this assertion in the new UTs that we add. Specifically, in the changes where new version is introduced.

We can have that conversation later, let's take this bug fix for now.